README.rst

```diff
@@ -65,9 +72,11 @@ docker-compose.yaml::yaml
         chadburn.job-exec.rss2email.schedule: "@every 5m"
         chadburn.job-exec.rss2email.command: "r2e run"
 
-The example docker-compose snippet sets up chadburn (or ofelia) job scheduler to run r2e periodically. For the labels to be picked up, it is required that the container keeps running which is ensured with the entrypoint. Python's Event().wait() was used as there is no shell and this method should use less cpu than sleeping or looping (with sleep). You can of course abandon this and just call r2e from any other job scheduler as this entrypoint only circumvents chadburn/ofelia limitations.
+The example docker-compose snippet sets up chadburn (or ofelia) job scheduler to run r2e periodically. 
```

Comment:
End of line whitespace.

Reply:
@plantroon 
The guide should either walk the user through the entire setup (e.g., how does chadburn get installed and invoked?) or leave out invocation details for users to figure out for themselves.

Reply:
Sorry, will fix